PrebidMobile/PrebidMobileRendering/ORTB/PBMORTBRendererConfig.m..m

```diff
@@ -0,0 +1,22 @@
+//
```

Comment:
I guess there is an issue with the extension here?
PrebidMobile/PrebidMobileRendering/ORTB/PBMORTBRendererConfig`.m..m`